og_migrate/migrations/d7_og_role.yml

```diff
@@ -0,0 +1,59 @@
+id: d7_og_role
+label: Og roles
+migration_tags:
+  - Drupal 7
+source:
+  plugin: d7_og_role
+process:
+  id:
+    -
+      plugin: concat
+      source:
+        - group_type
+        - group_bundle
+        - name
+      delimiter: '-'
+    -
+      plugin: og_role_machine_name
+    -
+      plugin: make_unique_entity_field
+      entity_type: og_role
+      field: id
+      migrated: true
+  label: name
+  weight: rid
+  # This could lead to bad data on the destination if something happened to the
```

Comment:
I forgot about this. We could follow-up with another deriver?

Reply:
Good for me, @amitaibu?

Reply:
:+1: